packages/@expo/config-plugins/src/android/Package.ts

```diff
@@ -237,8 +237,8 @@ export function setPackageInBuildGradle(config: Pick<ExpoConfig, 'android'>, bui
     return buildGradle;
   }
 
-  const pattern = new RegExp(`applicationId ['"].*['"]`);
-  return buildGradle.replace(pattern, `applicationId '${packageName}'`);
+  const pattern = new RegExp(`(applicationId|namespace) ['"].*['"]`, 'g');
+  return buildGradle.replace(pattern, `$1 '${packageName}'`);
```

Comment:
[@expo/config-plugins]: support 0.71 template transform. the namespace in build.gradle and the files in release build variants
cc @EvanBacon

Reply:
A test would be good here but otherwise this looks good to me.

Reply:
will follow up in separated pr 👍